cmd/meroxa/global/metrics.go

```diff
@@ -46,6 +46,10 @@ func NewPublisher() cased.Publisher {
 	if v := Config.GetBool("CASED_DEBUG"); v {
 		options = append(options, cased.WithDebug(v))
 	}
+
+	// TODO: Re-use HTTP Transform from our own Meroxa client
```

Comment:
Do you have a ticket to address this TODO?

Reply:
@dianadoherty Not yet! thanks for the call out.

Reply:
@dianadoherty https://github.com/meroxa/meroxa-go/issues/46

Reply:
Update: Right issue is https://github.com/meroxa/product/issues/16.